### PR TITLE
Add the docstring for the lr_cd.py

### PR DESCRIPTION
**Pull request by andyzhangstat**
Hi Teammates, 
Please review this docstring of algorithm function for milestone 1! Thank you!

### CODE REVIEW

**Comment by Jing-19** [comment]
Hi Andy, the docstring looks good to me! 

I noticed that your input is numpy array, but the data generated in the previous function is dataframe. you might want to convert the data type at the beginning of the function and change to input type to dataframe. There might be better way to achieve this, feel free to modify whichever it's easier. 

Also the visualization function will need the estimated y values from this function. Should we also return the estimated y here? Or do that in the visualization function? 

Thank you!

File: src/lr_cd/lr_cd.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    -------
+    ndarray
+        Optimized weights vector.
+
+
+
+    Examples
+    -------
+    >>> from lr_cd.lr_cd import coordinate_descent
+    >>> X = array([[ 1.        , -1.28162658],
```
Comment (Jing-19) [inline src/lr_cd/lr_cd.py]: It seems like the data generation function only return a two columns dataframe, but here we required a 2D array for X, and 1D array for y

**Review by Jing-19 — CHANGES_REQUESTED** [review]

File: src/lr_cd/lr_cd.py (hunk 1 above)
Comment (andyzhangstat) [reply src/lr_cd/lr_cd.py]: I use np array here to match the style of sklearn, that is, X is a np array and y is another np array. See the new python scripts for more details.

**Review by fohy24 — APPROVED** [review]
Everything looks in order to me. I changed the data generation to return 2 arrays too so it should have no problem reading. Thanks!

**Review by Jing-19 — APPROVED** [review]
Looks good to me! Thank you!

**Pull request merged**